CFC_DataCollector/recommender/recommendation_pipeline.py

```diff
@@ -14,26 +15,24 @@ def retrieve_alternative_trips(self, trip_list):
         return []
 
     def get_selected_user_utility_model(self, user_id):
-        return []
+        return UserUtilityModel.find_from_db(user_id)
 
     def recommend_trips(self, trip_id, utility_model):
         return []
```

Comment:
Sorry, I am not sure I remember the discussion.
How about something naive like adding an "original_trip" field to each alternative?
It should be pretty easy to then query the alternatives db for all alternatives for an original trip, and to lookup the original trip from the alternatives.